Week 1 Quiz Submission from ldeng11−stat6250

ldeng11-stat6250 opened the pull request.

prof-lankham commented:
Please review the "Files changed" summary and note the deletion of existing questions; please revise so that existing questions are not deleted.

prof-lankham commented:
This PR is now stale, so it's being closed.

Commit:
Week 1 Quiz Submission from ldeng11−stat6250

prof-lankham reviewed (changes requested):
Questions needed for all Problems/Recipes and should be relative to their content; also, questions should not be repeated

Commit:
Resolved merge conflict by incorpoerating both changes

Commit:
Merge remote-tracking branch 'upstream/master'

merge from the class main repo to sync my local fork copy

Commit:
Merge pull request #1 from stat6250/master

sync from class original

Commit:
Revised the week1 quiz

Sync the local repo with the class orginal first, revised the quiz based
on the instruction (add questions to each problems and recieps)

Commit:
Merge branch 'master' of https://github.com/ldeng11-stat6250/course-questions-wiki

prof-lankham committed:
Update README.md
